.github/workflows/builds.yml

```diff
@@ -49,6 +49,7 @@ jobs:
           source ${GITHUB_WORKSPACE}/.github/ci_setup.sh ${{ matrix.platform.name }}
           cmake --build build
       - name: Test
+        if: "matrix.platform.name == 'CPU'"
```

Comment:
:thinking: Not quite... Most of the SYCL tests should function on the CI nodes as well with `SYCL_DEVICE_FILTER=host`. So (as long as we don't add a HIP flavour for now) a

```
"matrix.platform.name != 'CUDA'"
```

would seem more appropriate.

Reply:
Fixed. :smile: